apps/librepcb/CMakeLists.txt

```diff
@@ -0,0 +1,105 @@
+# Enable Qt MOC/UIC/RCC
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTORCC ON)
+
+# Executable
+add_executable(librepcb
+    # When building on macOS, create a bundle
+    MACOSX_BUNDLE
+
+    # Sources
+    controlpanel/controlpanel.cpp
+    firstrunwizard/firstrunwizard.cpp
+    firstrunwizard/firstrunwizardpage_welcome.cpp
+    firstrunwizard/firstrunwizardpage_workspacepath.cpp
+    initializeworkspacewizard/initializeworkspacewizard.cpp
+    initializeworkspacewizard/initializeworkspacewizard_chooseimportversion.cpp
+    initializeworkspacewizard/initializeworkspacewizard_choosesettings.cpp
+    initializeworkspacewizard/initializeworkspacewizard_finalizeimport.cpp
+    initializeworkspacewizard/initializeworkspacewizardcontext.cpp
+    markdown/markdownconverter.cpp
+    projectlibraryupdater/projectlibraryupdater.cpp
+    main.cpp
+
+    # Headers
+    controlpanel/controlpanel.h
+    firstrunwizard/firstrunwizard.h
+    firstrunwizard/firstrunwizardpage_welcome.h
+    firstrunwizard/firstrunwizardpage_workspacepath.h
+    initializeworkspacewizard/initializeworkspacewizard.h
+    initializeworkspacewizard/initializeworkspacewizard_chooseimportversion.h
+    initializeworkspacewizard/initializeworkspacewizard_choosesettings.h
+    initializeworkspacewizard/initializeworkspacewizard_finalizeimport.h
+    initializeworkspacewizard/initializeworkspacewizardcontext.h
+    markdown/markdownconverter.h
+    projectlibraryupdater/projectlibraryupdater.h
+
+    # Forms
+    controlpanel/controlpanel.ui
+    firstrunwizard/firstrunwizard.ui
+    firstrunwizard/firstrunwizardpage_welcome.ui
+    firstrunwizard/firstrunwizardpage_workspacepath.ui
+    initializeworkspacewizard/initializeworkspacewizard.ui
+    initializeworkspacewizard/initializeworkspacewizard_chooseimportversion.ui
+    initializeworkspacewizard/initializeworkspacewizard_choosesettings.ui
+    initializeworkspacewizard/initializeworkspacewizard_finalizeimport.ui
+    projectlibraryupdater/projectlibraryupdater.ui
+
+    # Resources
+    ../../img/images.qrc
+)
+target_include_directories(librepcb PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/../../libs")
+target_link_libraries(librepcb PRIVATE
```

Comment:
I wonder if `-lproc` now also need to be specified here for Solaris, as you added it to the qmake files in https://github.com/LibrePCB/LibrePCB/pull/713 (?).

Reply:
Good point, I'll test whether the current version can be compiled on Solaris.

Reply:
Good catch. Fixed in https://github.com/LibrePCB/LibrePCB/pull/798/commits/f7b2fa77f56c9be6aa386c6f744f4a3dc76804c4 (ugh, that syntax 😄)